packages/components/src/List/Toolbar/ColumnChooser/hooks/columnChooserManager.hook.js

```diff
@@ -0,0 +1,154 @@
+import { useState } from 'react';
+import cloneDeep from 'lodash/cloneDeep';
+import invariant from 'invariant';
+import flow from 'lodash/flow';
+import { compareOrder } from '../service';
+
+export function isValueCorrect(value, collectionLength) {
+	return Number.isInteger(value) && (value <= collectionLength || value < 0);
```

Comment:
I don't understand the last condition, is it a mistake ?
Here negative value returns true to isValueCorrect()

Reply:
Resolved